<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(228, 228, 228);

background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>641</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>441</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(88, 81, 139);
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0.04, stop:0 rgba(88, 81, 139, 234), stop:1 rgba(99, 128, 202, 255));</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>861</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 177, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(90, 120, 199, 246), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 italic 14pt &quot;Segoe UI Black&quot;;</string>
     </property>
     <property name="text">
      <string>F A C U L T Y   S E L E C T I O N</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>501</width>
       <height>591</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>501</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(227, 227, 227);</string>
     </property>
     <widget class="QPushButton" name="button1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>421</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(147, 176, 254);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(183, 186, 195, 255), stop:1 rgba(255, 255, 255, 255));
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Engineering</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>421</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(147, 176, 254);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(183, 186, 195, 255), stop:1 rgba(255, 255, 255, 255));
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Environmental Design</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>421</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(147, 176, 254);
font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(183, 186, 195, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>Education</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
